android/src/main/java/com/reactnativecommunity/viewpager/ViewPagerFragment.java

```diff
@@ -4,22 +4,58 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.widget.FrameLayout;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 
+import java.lang.ref.WeakReference;
+
+import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
+
 public class ViewPagerFragment extends Fragment {
-    View view;
+    private final int mPosition;
+    private WeakReference<View> mReactViewRef;
 
-    public ViewPagerFragment(View child) {
-        view = child;
+    public ViewPagerFragment(int position, View child) {
+        mPosition = position;
+        mReactViewRef = new WeakReference<>(child);
     }
 
-
     @Nullable
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-        return view != null ? view : new View(getContext());
+        FrameLayout rootView = new FrameLayout(inflater.getContext());
```

Comment:
Why react view is wrapped into framelayout ? 
Why we cannot directly return react view ?

Reply:
Originally used the react view directly, however I switched in https://github.com/callstack/react-native-viewpager/pull/279/commits/610dcac0b2ab00cd0b4aa2b5225c5a347e32bd18 because I could not determine any other way to solve the blank page bug.  Android would get stuck on a white screen if the user paged fast enough for the screen not to be rendered before arriving.  I tried using `notifyItem*()` methods, but they did not help.

I would prefer using the react view directly, if there is some way to fix the blank screen bug.